sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormTable.cs

```diff
@@ -33,11 +34,10 @@ internal ExtractedTable(DataTable_internal table, ReadResult_internal readResult
         // TODO: Handling column-span?
         // https://github.com/Azure/azure-sdk-for-net/issues/9975
 
-
         /// <summary>
         /// </summary>
 #pragma warning disable CA1822 // Mark as static
-        public ExtractedTableCell this[int row, int column]
+        public FormTableCell this[int row, int column]
```

Comment:
the set for this property is marked as internal in the original PR

Reply:
It should be internal until we implement it.

Reply:
You're right. I thought it was `internal` in the original file and left like that by accident, since the new API expects it to be `public`. However, after taking a looking here it seems Anne intentionally changed it from `public` to `internal` in her PR. I'm assuming it was so we don't end up shipping it in a non-functional state by accident. I'll get it updated before merging the PR.

Reply:
I didn't refresh the page before replying to Mariana, so I wrote my comment before reading Anne's. This probably made my reply a bit awkward, but I think we all agree in keeping it `internal` then.